ledger/types/balances_json.go

```diff
@@ -0,0 +1,108 @@
+package types
+
+import (
+	"encoding/json"
```

Comment:
We use the `"github.com/goccy/go-json"` package in other parts of the library as it's more performant than the standard `json` library